File: TestDemoA/app/src/main/java/com/jieleo/testdemoa/MainActivity.java
package com.jieleo.testdemoa;

import android.content.Intent;
import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void ToB(View view){
        Intent intent =new Intent("TestDemoB.com", Uri.parse("value://123"));
        intent.putExtra("key","我是A中的值");
        startActivity(intent);
    }
}

File: TestDemo/app/src/main/java/com/lc/duoweidu/project/MainActivity.kt
package com.lc.duoweidu.project

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.lc.duoweidu.project.face.PermissionAcitivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var intent:Intent
//        intent= Intent(this,PermissionAcitivity::class.java)
        intent= Intent(this,PostTestActivity::class.java)
        startActivity(intent)

    }
}
